keps/sig-auth/267-kubelet-server-certificate-bootstrap-rotation/README.md

```diff
@@ -0,0 +1,438 @@
+- [ ] Documentation created: https://github.com/kubernetes/website/issues/30575
+
+### Upgrade / Downgrade Strategy
+
+Feature exists for a long time, no risk for Upgrade/Downgrade.
+
+### Version Skew Strategy
+
+Feature exists for a long time, no risk for version skew.
+
+## Production Readiness Review Questionnaire
+
+### Feature Enablement and Rollback
+
+Feature exists for a long time, no risk for Enablement and Rollback.
+
+###### How can this feature be enabled / disabled in a live cluster?
+
+Configuration settings must be used - command line argument
+(`--rotate-server-certificates`: deprecated) or Node config flag
+`serverTLSBootstrap`.
+
+`serverTLSBootstrap` or `--rotate-server-certificates` can be set to false and the feature will be disabled.
+
+In beta, it is also possible to turn the feature flag off. This is not possible in GA.
+
+###### Does enabling the feature change any default behavior?
+
+No, unless kubelet uses special config.
+
+###### Can the feature be disabled once it has been enabled (i.e. can we roll back the enablement)?
+
+Yes
+
+###### What happens if we reenable the feature if it was previously rolled back?
+
+Kubelet will not start unless configuration will be changed to useself-signed
+certificate.
+
+###### Are there any tests for feature enablement/disablement?
+
+N/A
+
+### Rollout, Upgrade and Rollback Planning
+
+N/A
+
+###### How can a rollout or rollback fail? Can it impact already running workloads?
+
+N/A
+
+
+###### What specific metrics should inform a rollback?
+
+N/A
+
+###### Were upgrade and rollback tested? Was the upgrade->downgrade->upgrade path tested?
+
+N/A
+
+###### Is the rollout accompanied by any deprecations and/or removals of features, APIs, fields of API types, flags, etc.?
+
+N/A
+
+### Monitoring Requirements
+
+N/A
+
+###### How can an operator determine if the feature is in use by workloads?
+
+Yes, using metrics 
+
+###### How can someone using this feature know that it is working for their instance?
+
+N/A
+
+###### What are the reasonable SLOs (Service Level Objectives) for the enhancement?
+
+Metrics can be used to determine the speed of cert update.
+
+###### What are the SLIs (Service Level Indicators) an operator can use to determine the health of the service?
+
+<!--
+Pick one more of these and delete the rest.
+-->
+
+- [ ] Metrics
+  - Metric name:
+  - [Optional] Aggregation method:
+  - Components exposing the metric:
+- [ ] Other (treat as last resort)
+  - Details:
+
+###### Are there any missing metrics that would be useful to have to improve observability of this feature?
+
+<!--
+Describe the metrics themselves and the reasons why they weren't added (e.g., cost,
+implementation difficulties, etc.).
+-->
+
+### Dependencies
+
+- Certificate Signing Request API. This API is GA.
+- Configuration to approve signing requests.
+
+This feature only works when properly configured.
+
+###### Does this feature depend on any specific services running in the cluster?
+
+No
```

Comment:
Doesn't this depend on a CSR approver?

Reply:
What usually approves CSRs? I'm not very well versed in this area so I'm interested to learn more.